scratch/copy-mk.py

```diff
@@ -89,7 +89,7 @@ async def async_main(args) -> int:
         for n, grp in enumerate(mcast_groups[1:]):
             if grp == mcast_groups[0]:
                 # MK is not using a full power-of-2 size array. There are dummies from here onwards.
-                print("Dummy antenna found starting at ", (n + 1) // 2)
+                print(f"Dummy antenna found starting at {(n + 1) // 2}")
```

Comment:
Is there something inherently preferable about an f-string?

Reply:
I was undecided between doing it this way and just deleting the trailing space of the string literal, but I figure with f-strings it's harder to screw up the spacing.

Reply:
That's a fair point.